app/src/main/java/com/example/habitsmasher/ui/dashboard/EditHabitFragment.java

```diff
@@ -112,17 +121,29 @@ public void onClick(View v) {
             }
         });
 
+        //set up days of the week buttons
+        setListenersForDaysOfTheWeek();
+
+        return view;
+    }
+
+    /*
+     * Sets up the buttons for the days of the week, including setting the initial states.
+     */
+    private void setListenersForDaysOfTheWeek(){
```

Comment:
Minior: Since this method is doing more than just setting the listeners, perhaps it can be renamed to something like initializeDaysOfWeeksButtons()?

Reply:
Yep, will do.